site2/docs/client-libraries-python.md

```diff
@@ -39,7 +39,7 @@ $ sudo python setup.py install
 
 ## API Reference
 
-The complete Python API reference is available at [api/python]({{site.baseUrl}}/api/python).
+The complete Python API reference is available at [api/python](/api/python).
```

Comment:
do you need to fix the links for other versions?

Reply:
Done